traits/tests/test_view_elements.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2019 by Enthought, Inc.
+# All rights reserved.
+
+import unittest
+
+from traits.api import HasTraits, Int, TraitError
+from traits.testing.optional_dependencies import requires_traitsui
+
+
+@requires_traitsui
+class TestViewElements(unittest.TestCase):
+    def setUp(self):
+        import traitsui.api
```

Comment:
Off-topic for this PR, but I think the `traitsui` that `optional_dependencies` exports should be adjusted so that `traitsui.api` is already valid. Then you'd just do `import requires_traitsui, traitsui` above and be ready to go without any further imports.

Reply:
Done in #616